app.js

```diff
@@ -177,6 +178,14 @@ History.prototype.toJSON = function () {
     return this.facts.reverse().toJSON();
 };
 
+function buildFinalTRpath(baseURL, thisVersionURL) {
+
+    var w3URL = /https?:\/\/(www\.)?w3\.org\/tr/i;
```

Comment:
I was just trying to be as flexible as possible with our inputs. The registration form will accept something like `http://WWW.w3.oRg...` as the URL, right? So it should still work further along the process, I think. That's the good thing about regexes&hellip;